### ISSUE TITLE
Compilation error when using WebPack: The "to" argument must be of type string

### ISSUE THREAD

**Issue by hammadj** [react-cosmos/react-cosmos#1208]
### What's wrong?

When running react-cosmos with an Expo React Native app and custom webpack override, I get the following error: 
```
webpack building...
[Cosmos] Generated cosmos.userdeps.js
webpack built 6429cdf9d84460757613 in 661ms
internal/validators.js:118
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "to" argument must be of type string. Received type number (1590631463516)
```

The culprit is this line: https://github.com/react-cosmos/react-cosmos/blob/733e4992f276413a562b430cf406a5dfb0225229/packages/react-cosmos/src/plugins/webpack/devServer.ts#L63

When I change it to 
```js
const relFilePath = path.relative(process.cwd(), filePath.toString());
```
it works perfectly fine

### User info

- Cosmos version: 5.3.0
- Cosmos config:
```json
{
  "$schema": "http://json.schemastore.org/cosmos-config",
  "webpack": {
    "overridePath": "./webpack.override.js"
  }
}
```

- If applicable: webpack config:
```js
const { createAllLoaders } = require('@expo/webpack-config/loaders');

const { withUnimodules } = require('@expo/webpack-config/addons');

module.exports = webpackConfig => {
  const expoEnv = {
    projectRoot: __dirname,
    babel: {
      dangerouslyAddModulePathsToTranspile: [
        // Ensure that all packages starting with @unloan are transpiled.
        '@unloan',
        'static-container'
      ],
    },
  };

  // Add Expo support...
  const configWithExpo = withUnimodules(webpackConfig, expoEnv);

  configWithExpo.module.rules = createAllLoaders(expoEnv);
  
  return configWithExpo;
};
```

**Comment by skidding** [comment]
Hi @hammadj,

So the invalid hook is called with a timestamp instead of a file path. That odd, as the `changeTime` is supposed to be the 2nd argument: https://webpack.js.org/api/compiler-hooks/#invalid.

What's your webpack version?

**Comment by hammadj** [comment]
@skidding Im using webpack@4.39.0

**Comment by skidding** [comment]
@hammadj I pushed a fix to handle non-string file paths in ddb94b10fe2b1909560385cec3a1c61bba42f21a.

Please try `react-cosmos@5.4.0-alpha.0` and let me know if the error goes away.

**Comment by hammadj** [comment]
@skidding awesome, works great now
